src/requirements.txt

```diff
@@ -29,21 +29,22 @@ jmespath==1.0.1 ; python_version >= '3.7'
 mako==1.2.4 ; python_version >= '3.7'
 markupsafe==2.1.2 ; python_version >= '3.7'
 marshmallow==3.19.0 ; python_version >= '3.7'
-oic==1.5.0
+oic==1.6.0
 orderedmultidict==1.0.1
-packaging==23.0 ; python_version >= '3.7'
-phonenumberslite==8.13.6
-psycopg2-binary==2.9.5
+packaging==23.1 ; python_version >= '3.7'
+phonenumberslite==8.13.13
+psycopg2-binary==2.9.6
 pycparser==2.21
-pycryptodomex==3.17
+pycryptodomex==3.18.0
+pydantic==1.10.8 ; python_version >= '3.7'
 pyjwkest==1.4.2
 python-dateutil==2.8.2 ; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
-python-dotenv==0.21.1 ; python_version >= '3.7'
-requests==2.28.2
-s3transfer==0.6.0 ; python_version >= '3.7'
-setuptools==67.2.0 ; python_version >= '3.7'
+python-dotenv==1.0.0 ; python_version >= '3.8'
```

Comment:
🧵 Major version bump

Reply:
https://pypi.org/project/python-dotenv/#history

Reply:
https://github.com/theskumar/python-dotenv/blob/main/CHANGELOG.md#100

Reply:
Support for Python 3.7 was dropped, which I assume is the reason for the major version bump

Reply:
Our docker compose uses Python 3.10, so we should be good.